docs/modules/ROOT/pages/interfaces.adoc

```diff
@@ -0,0 +1,177 @@
+= Interfaces and Dispatchers
+
+This section describes the interfaces OpenZeppelin Contracts for Cairo offer, and explains the design choices behind them.
+
+Interfaces can be found in the module tree under the `interface` submodule, such as `token::erc20::interface`. For example:
+
+```javascript
+use openzeppelin::token::erc20::interface::IERC20Dispatcher;
+```
+
+or
```

Comment:
This or is confusing to me, I think we are talking about contract interfaces but then a dispatcher trait is presented. I think we should split the ideas (Interface vs Dispatchers vs ABI)

Reply:
removed `Dispatcher`, it was a mistake. this "or" was just to show another example.